src/olympia/discovery/serializers.py

```diff
@@ -48,4 +49,10 @@ def to_representation(self, instance):
                 '{start_sub_heading}', '<span>').replace(
                 '{end_sub_heading}', '</span>').replace(
                 '{addon_name}', addon_link)
+
+        if (data['description'] is None and
+                instance.addon.type == amo.ADDON_EXTENSION and
+                instance.addon.summary):
+            data['description'] = (
+                u'<blockquote>%s</blockquote>' % instance.addon.summary)
```

Comment:
Aaand this is where we start having l10n issues, since the add-on is not guaranteed to have a translation in the user language :/ It's a problem we already have elsewhere on the site, but I was hoping we'd continue to avoid it in the disco pane, which is, in a way, part of Firefox itself.

Reply:
Unavoidable problem really, unless we want to start ignoring recommendations that don't have a translation in that language.  Which is totally doable, but is going to seriously limit the usefulness to non-English speakers.

Reply:
This is a known and accepted issue that we will potentially have a mix of translated and untranslated content.